spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Users::SessionsController, type: :controller do
+  describe "POST #create" do
+    before :each do
+      @request.env["devise.mapping"] = Devise.mappings[:user]
+      @user = FactoryGirl.create :user
+    end
+
+    context "when the credentials are correct" do
+      before :each do
+        credentials = { email: @user.email, password: '123456' }
+        post :create, params: { session: credentials }
+      end
+
+      it "returns the user record corresponding to the given credentials" do
+        @user.reload
+        response_token = JSON.parse(response.body)["authentication_token"]
```

Comment:
replace this with let(:response_token) { JSON.parse(response.body)["authentication_token"] } on top